site/content/docs/developer-guide/release/submitting-to-krew.md

```diff
@@ -19,7 +19,7 @@ existing plugins.
 1. Make sure to extract the LICENSE file during the plugin installation.
 1. Tag a git release with a [semantic
    version](https://semver.org) (e.g. `v1.0.0`).
-1. [Test your plugin installation locally]({{< ref "../installing-locally.md" >}}).
+   > **Note:** the 'v' is required and will be linted against
```

Comment:
I think we don't require v in git tag so I think instead of updating this file, the edit you made on sample yaml is sufficient.

Reply:
Ok, doesn't seem to be required but "highly encouraged".

https://github.com/kubernetes-sigs/krew/blob/eda76d9eb403f290d3f25b7b2930b51f170a1312/site/content/docs/developer-guide/release/plugin-updates.md?plain=1#L16